react-client-app/src/components/ScanReportTbl.jsx

```diff
@@ -19,6 +19,8 @@ const ScanReportTbl = (props) => {
     const [authorFilter, setAuthorFilter] = useState("All");
     const [title, setTitle] = useState("Scan Reports Active");
     const [expanded, setExpanded] = useState(false);
+    const statuses = ["UPINPRO", "UPCOMPL", "UPFAILE", "PENDING", "INPRO25", "INPRO50", "INPRO75", "COMPLET", "BLOCKED"]
```

Comment:
Is there any way to grab these (and the contents of `mapStatus()` below) from the django model rather than hardcoding them here? Better for us to only have to change it in one place.